examples/resources/rafay_aks_cluster_v3/resource.tf

```diff
@@ -0,0 +1,85 @@
+resource "rafay_aks_cluster_v3" "demo-terraform" {
+  metadata {
+    name    = "aks-v3-tf-2401202303"
+    project = "defaultproject"
+  }
+  spec {
+    type          = "aks"
+    blueprint_config {
+      name = "default-aks"
+      version = "1.21.0"
+    }
+    cloud_credentials = "poornima-aks-cred-dev"
```

Comment:
can you change the names in the example to more generic ones? Customers see this in the doc. Refer other examples